source/slang/slang-type-layout.cpp

```diff
@@ -703,6 +847,28 @@ struct CUDALayoutRulesImpl : DefaultLayoutRulesImpl
         // Conform to CUDA/C/C++ size is adjusted to the largest alignment
         ioStructInfo->size = _roundToAlignment(ioStructInfo->size, ioStructInfo->alignment);
     }
+
+    ObjectLayoutInfo GetDescriptorHandleLayout(
+        LayoutResourceKind varyingKind,
+        Type* elementType,
+        const TypeLayoutContext& context) override
+    {
+        // For CUDA targets, DescriptorHandle<T> has the layout of T
+        // Determine ShaderParameterKind from the element type
+        ShaderParameterKind paramKind = _getShaderParameterKindForResourceType(elementType);
+
+        // Get the object layout to get size
+        auto objLayout = context.rules->GetObjectLayout(paramKind, context.objectLayoutOptions);
+
+        // objLayout can have multiple layoutInfos, so we need to return the size from each
+        ObjectLayoutInfo result;
```

Comment:
isn't `objLayout` already a ObjectLayoutInfo? If so we should be able to just return it directly here?